scripts/check-diff.sh

```diff
@@ -583,7 +583,7 @@ function install_wp {
 
 	# Download `wp-includes` folder from the WordPress Core SVN repo to include built internal dependencies.
 	local SVN_CORE_URL=${SVN_URL/develop/core}
-	svn export -q "${SVN_CORE_URL}wp-includes" "$WP_CORE_DIR/src/wp-includes"
+	svn export -q --force "${SVN_CORE_URL}wp-includes" "$WP_CORE_DIR/src/wp-includes"
```

Comment:
Alternatively, this could also `rsync` the exported files on top of the the existing directory. Maybe this is what `--force` does in practice.

Reply:
It appears not, `--force` replaces the folder. `rsync` could work here as well.